aws_embedded_metrics/constants.py

```diff
@@ -14,3 +14,4 @@
 DEFAULT_NAMESPACE = "aws-embedded-metrics"
 MAX_DIMENSIONS = 9
 MAX_METRICS_PER_EVENT = 100
+MAX_DATAPOINTS_PER_EVENT = 100
```

Comment:
Consider renaming to `MAX_DATAPOINTS_PER_METRIC`.

Reply:
Updated!